src/openam/templates/run-configmap.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  labels:
+    app: {{ template "openam.name" . }}
+    chart: {{ .Chart.Name }}-{{ .Chart.Version }}
+    component: "{{ template "openam.name" . }}"
+    heritage: {{ .Release.Service }}
+    release: {{ .Release.Name }}
+  name: {{ template "openam.fullname" . }}-run
+data:
+  run_me.sh: |
+    #!/usr/bin/env bash
+
+    # Instance dir does not exist? Then we need to run setup
+    if [ ! -f ${BASE_DIR}/install.log ] ; then
+      echo "OpenAM config not found. Configuring.."
+      /bin/configure.sh &
```

Comment:
This is a roundabout way to create an `install.log`. If you're checking for the existence of the install log in this script, i would have the configure.sh script output to stdout and then redirectr to that log file here in this script.

Reply:
Configure.sh is configuring the default realm so that manually configuration using UI is not required. Since this configuration data is stored in PV and configuration is one time job, above check is to make sure we do not overwrite already configured data. Configure.sh creates install.log if default realm is configured successfully.